.github/ISSUE_TEMPLATE/engineering-onboarding.md

```diff
@@ -86,6 +86,7 @@ _Remember to review *all* setup scripts before executing them :)_
   - [ ] [Back-end/Platform technologies](https://github.com/artsy/README/blob/master/practices/platform.md)
   - [ ] iOS/React Native? #TODO
 - [ ] Reading: [Best of Our Blog](https://github.com/artsy/README/blob/master/resources/blog.md)
+- [ ] Watch some of our [Lunch & Learns](https://github.com/artsy/README/blob/master/resources/lnl.md)
```

Comment:
Nice list! 👍 

Can you move this down one item so that it’s not interleaved with all tech related links?